src/main/java/org/broadinstitute/hellbender/tools/ClipReads.java

```diff
@@ -505,7 +505,7 @@ public SeqToClip(String name, byte[] bytez) {
         public long nRangeClippedBases = 0;
         public long nSeqClippedBases = 0;
 
-        HashMap<String, Long> seqClipCounts = new HashMap<>();
+        SortedMap<String, Long> seqClipCounts = new TreeMap<>();
```

Comment:
the `seqClipCounts` are better organized when sorted by the String key than just in arbitrary order. So sorted map is better.